float64 sound_pressure_level

